src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -35,6 +35,12 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.io.IOUtils;
```

Comment:
You've reorganized the imports here, these lines have shifted place. Don't do this, it makes the patch files larger for no reason.

Reply:
This is a forge class. Imports should be added but not organized, its entirely up to the users IDE as to how they are ordered.

Reply:
So I don't have to organize them?

Reply:
No, don't organize them.